### PR TITLE
Add `expect_zero` back for filestore_utils.cp

### PR DESCRIPTION
**Pull request by zchcai**
This fix #499. There will be no `ERROR:root:Executed command: "cp ...`
This pattern follows the pattern in old implementation, https://github.com/google/fuzzbench/tree/ec27e1711cd4c4904e3c0b853f046c1d98b7b189
https://github.com/google/fuzzbench/blob/ec27e1711cd4c4904e3c0b853f046c1d98b7b189/experiment/measurer.py#L398-L402

https://github.com/google/fuzzbench/blob/ec27e1711cd4c4904e3c0b853f046c1d98b7b189/experiment/measurer.py#L536-L541

### CODE REVIEW

**Comment by zchcai** [comment]
Related to #251, #422.

File: experiment/measurer.py [hunk 1]
```diff
@@ -540,9 +537,8 @@ def measure_snapshot_coverage(fuzzer: str, benchmark: str, trial_num: int,
     if not os.path.exists(corpus_archive_dir):
         os.makedirs(corpus_archive_dir)
 
-    try:
-        filestore_utils.cp(corpus_archive_src, corpus_archive_dst)
-    except subprocess.CalledProcessError:
+    if filestore_utils.cp(
+            corpus_archive_src, corpus_archive_dst, expect_zero=False)[0] != 0:
```
Comment (inferno-chromium) [inline experiment/measurer.py:541]: nit: "!= 0" can be removed.

File: common/filestore_utils.py [hunk 2]
```diff
@@ -37,12 +37,13 @@ def get_impl():
     return local_filestore
 
 
-def cp(source, destination, recursive=False, parallel=False):  # pylint: disable=invalid-name
+def cp(source, destination, recursive=False, parallel=False, expect_zero=True):  # pylint: disable=invalid-name
     """Copies |source| to |destination|."""
```
Comment (lszekeres) [inline common/filestore_utils.py:41]: nit: consider extending the docstring to describe all parameters.

File: common/gsutil.py [hunk 3]
```diff
@@ -28,14 +28,14 @@ def gsutil_command(arguments, expect_zero=True, parallel=False):
     return new_process.execute(command + arguments, expect_zero=expect_zero)
 
 
-def cp(source, destination, recursive=False, parallel=False):  # pylint: disable=invalid-name
+def cp(source, destination, recursive=False, parallel=False, expect_zero=True):  # pylint: disable=invalid-name
     """Executes gsutil's "cp" command with |cp_arguments|."""
```
Comment (lszekeres) [inline common/gsutil.py:32]: nit: the docstring seems to be out of date. could you please update it? thanks!

**Pull request merged**